Makefile

```diff
@@ -124,21 +124,14 @@ PINNED_DEPENDENCIES := \
 # Code coverage & test report output files.
 TEST_OUTPUT_ROOT        := ./.testoutput
 NEW_COVER_PROFILE       = $(TEST_OUTPUT_ROOT)/$(shell xxd -p -l 16 /dev/urandom).cover.out   # generates a new filename each time it's substituted
-SUMMARY_COVER_PROFILE  := $(TEST_OUTPUT_ROOT)/summary.cover.out
 NEW_REPORT              = $(TEST_OUTPUT_ROOT)/$(shell xxd -p -l 16 /dev/urandom).junit.xml   # generates a new filename each time it's substituted
+COVERPKG 				= $(shell go list ./... | paste -sd "," -)
```

Comment:
Just listing all packages here. Anything we don't want to be reported is filtered out by Codecov.